docs/programming-languages/javascript-and-node-js.md

```diff
@@ -174,3 +172,37 @@ for details on pre-installed browsers and testing tools on Semaphore.
 [macos-javascript]: https://docs.semaphoreci.com/ci-cd-environment/macos-mojave-xcode-11-image/#javascript-via-node-js
 [docker-env]: https://docs.semaphoreci.com/ci-cd-environment/custom-ci-cd-environment-with-docker/
 [node-docker-image]: https://hub.docker.com/r/semaphoreci/node
+
+## Connecting your Node.js application to the test Database
+
+Create an example configuration file at `project_name/config/config.js` with the following:
+``` javascript
+// config/config.js
+
+module.exports = {
+  postgres: {
+    database: process.env.DATABASE_NAME,
+    username: process.env.DATABASE_USERNAME,
+    password: process.env.DATABASE_PASSWORD
+  },
+}
+```
+Define the [secrets](/guided-tour/environment-variables-and-secrets/#managing-sensitive-data-with-secrets) for `DATABASE_NAME`, `DATABASE_USERNAME` and `DATABASE_PASSWORD`
```

Comment:
This also doesn't have to be a secret. Values for name, username and password are usually not sensitive.

We can say:

> Start a Postgres database in your job with [sem-service](docs):
>
> export DATABASE_USERNAME="test-user"
> export DATABASE_PASSWORD="keyboard-cat"
> export DATABASE_NAME="test-db"
>
> sem-service start postgres --username="$DATABASE_USERNAME" --password="$DATABASE_PASSWORD" --db="$DATABASE_NAME"